tests/console/yast2_ftp.pm

```diff
@@ -85,18 +85,25 @@ sub run {
 
     # start yast2 ftp configuration
     type_string "yast2 ftp-server; echo yast2-ftp-server-status-\$? > /dev/$serialdev\n";
-    assert_screen 'ftp-server';                 # check ftp server configuration page
-    send_key 'alt-w';                           # make sure ftp start-up when booting
-    assert_screen 'ftp_server_when_booting';    # check service start when booting
+    assert_screen 'ftp-server';    # check ftp server configuration page
+    if (is_sle('<15') || is_leap('<15.0')) {
```

Comment:
Is is the case that we have different conditions for different YaST modules? I see 3 variations, wonder if it's the case.

Reply:
wrong paste, good catch! We have two cases, `is_sle('<15') || is_leap('<15.0')` and another one that also include tw, as in samba is not yet for that product.